charts/csi-secret-provider-class/values.yaml

```diff
@@ -14,3 +14,34 @@ secretObjects: []
 # secretObjectsTpl -- template to generate secretObjects
 # @default -- raw values from `secretObjects`
 secretObjectsTpl: "{{ .Values.secretObjects | toYaml }}"
+
+# Creates a dummy Deployment which uses the csiVolume. This is a hack, in case your Chart doesn't support defining extraVolumes
```

Comment:
Maybe we should add another strongly worded sentence here that tells people not to use this unless they tried fixing the underlying issue and can prove this with a PR or support ticket for the affected chart.

Reply:
```suggestion
# Creates a dummy Deployment which uses the csiVolume. This is a hack, in case your Chart doesn't support defining extraVolumes
# DO NOT USE THIS unless strictly needed. Always try to use extraVolume Bindings inside your already existing Chart or application to make csi-secret-store manage the secret
```

Reply:
Something like this?

Reply:
yeah & with an additional mention of the upstream work that we expect you to put in before you are allowed to use it

Reply:
Better?

Reply:
yes, thanks!

Reply:
Did you end up changing this?

Reply:
Just now.

Reply:
Now checks are failing... ngg

Reply:
Checks are passing @hairmare